Source/JavaScriptCore/interpreter/CallFrameInlines.h

```diff
@@ -78,6 +78,14 @@ inline JSGlobalObject* CallFrame::lexicalGlobalObject(VM& vm) const
     return jsCallee()->globalObject();
 }
 
+#if ENABLE(WEBASSEMBLY)
+inline Wasm::Instance* CallFrame::wasmInstance() const
+{
+    ASSERT(callee().isWasm());
+    return bitwise_cast<Wasm::Instance*>(const_cast<CallFrame*>(this)->uncheckedR(CallFrameSlot::codeBlock).asanUnsafePointer());
```

Comment:
I almost think we should have a second enum mirroring `CallFrameSlot` that has all the wasm slots. But we can do that another day. Alternatively, we could rename this to `CallFrameSlot::jsCodeBlockOrWasmInstance` but I think having two enums is clearer.